diagnostics/app/model/diagnostics/commercial/RedisReport.scala

```diff
@@ -42,7 +44,10 @@ class ExpiredKeyEventSubscriber(client: RedisClient, system: ActorSystem) extend
         reportData <- redisClient.get(RedisReport.dataKeyFromId(id))
       } {
         log.logger.info(s"writing report to s3 bucket, view id: $id")
-        S3CommercialReports.putPublic(s"commercial-client-logs/$id", reportData, "text/plain")
+
+        if (Configuration.environment.isProd) {
```

Comment:
Why do this here? It's not really uploading to S3 anymore. Maybe do it in the `callback`?

Why do you even need this anyway? Does it listen on `CODE`?